test/e2e/dynamic_provisioning_test.go

```diff
@@ -826,15 +825,14 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 			CSIDriver:              testDriver,
 			Volume:                 volume,
 			Pod:                    pod,
-			ResizeOffline:          false,
+			ResizeOffline:          isWindowsCluster,
 			StorageClassParameters: map[string]string{"skuName": "Standard_LRS"},
 		}
 		test.Run(cs, ns)
 	})
 
 	ginkgo.It("should create a block volume on demand and dynamically resize it without detaching [disk.csi.azure.com] ", func() {
 		skipIfUsingInTreeVolumePlugin()
-		skipIfNotDynamicallyResizeSuported()
```

Comment:
Shouldn't this test case be skipped on Windows? I don't believe block volumes are supported on Windows.